app/assets/javascripts/hyrax/admin/collection_type/settings.es6

```diff
@@ -0,0 +1,52 @@
+export default class {
+  constructor(element) {
```

Comment:
Please add jsdoc showing that element is a `jQuery` object and document that it must contain a checkbox within it.

Reply:
still need doc here.